package.json

```diff
@@ -55,18 +55,25 @@
         "stylelint-formatter-pretty": "^2.1.1",
         "stylelint-no-unsupported-browser-features": "^4.0.0",
         "stylelint-selector-bem-pattern": "^2.1.0",
-        "stylelint-webpack-plugin": "^2.1.1"
+        "stylelint-webpack-plugin": "^2.1.1",
+        "nx": "14.1.9",
+        "@nrwl/nx-cloud": "latest"
     },
     "scripts": {
+        "build:all": "nx run-many --target=build --all",
```

Comment:
do we need `--all` here? based on the documentation the option is deprecated

Reply:
yess you are right, by default it will run the command on all workspaces